pkg/comp-functions/functions/vshnnextcloud/files/sample-key.rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEpQIBAAKCAQEAvIImAilKC6vry7nRYi+1cmbafYd2OtMlaJNyw86Rwu4COv1B
+OCg01zwiGnLqbC3WWcfxrbuwOowRTYxBKQ8GdaoqkrXrgiLKFfAHtxDWpmuZZ6/6
+AzqneaTDV3yF1953ID4y0kFg8GxGVmc+eeIcUKKjYjcG/3yY5llefzFxC4ABNlpK
+t19lmKzHIdPye0A7QUvSiPv3onfa1d+oPB8fB4o+9wV2k2ZbGhoKGCUqGdCcWZa8
+SQf6UoqWjRfJTdfI9j5XwLQWtuZqV+GRe6fIfDWQPhhslZAAPUj9q8jMOSZx16gX
+7ioq4nNLk++0lVpLSqr5rZUKlkQ8yLbheI3S3QIDAQABAoIBAQCX03spYnfwzpcr
+jo3Ftwc9DPoNAH2JA2atw5Q7gLSFe4zwVemhgUCN2jDdfJCycHij9xYAB5r4jLei
+7cPNffyF6HhBkaQNPFfnM1wDZdx8TU1Q5btHvoYEw19Rp0llhBgOMAperamV+p7s
+GFuJb0O+7j8Ja6iHPaMoY/JoHzodN6l2xa2QDR71jmhZkzOGcg8qOPh9cCVgfqQh
+pUbkg/XPaFRfO7ufqzuVmF/gu3oFes6037YVq6rpl+xWkwfavPtXqLJSI+FVDWww
+QJys0jcJH5rgYHUH9Vlmf5mbuWgsV4VoVkLGSouvD2Z/AZnQvlDygdB0kyMrzzPf
+wO9LvrYBAoGBAObG5RynntSAPR03x9lLcto0njNldj1fhSGxxdPp6EakdaIMmT8E
+ruae6hvCY+NgjxlPPtHh9ozvFvBhJ4JJLdW+FzyOsGGOvzzhHsxJ4g7E4wZ/9Jvz
+syoDOBR7cEut4rwh4In8qd2/KdZDuX+2pox7xmMNOd74SRAUkjaRJHCpAoGBANEc
+lfu/2cqVYj/MHZKBVP7Trk6PRBCgW7FNVoxW0UubCeLLxT+SkDZ5sZKlZPVlHWhs
+5vSenuNQWTyQ+dRqN8WlZJgT4DpZ+lG2y7WIJU8eCBnYPGNBC8cdd0OJfh5uPmyw
+mfMmSBwgSM99zQRICuGRk3uXQnTMX76sSEpPGA0VAoGBAMGVsFU0KqD5HcrnosYd
+ay+tmdNlViUvfl8AYAFeolC33lFcHiTlyc19GsbEbsQqmH9ToU+wLACXAvgLl3fL
+6R32QkmzLDP56WOpOAuIIo/fCs3GAivb5rvkTWOpFE73g03rYQ5/GwL/O5oX34BJ
+eDY6EfW+/3+RaMCLcQdDA31ZAoGAJWU/TS4Z/KiKUW5nO3P1hzzDxzjHq1GCFEsI
+V0CPXT30nqJiO+Qx9gv9PYRmcZwLjkkrZEOZln0f+DzU8K+uGaJiQvALAWRl54cR
+Fv2p7mQofra01esj+A/E7EcCTt7YOwe6AEmwS5dabgDJUSKxD3XBMjDyqYCC/s5A
+f0l1u6ECgYEAt+QSB/6UVjXIRxhdTPsJ0eZEsZaNNNAhLnSVDaE0kvZKPU1vHgsv
+4Gb0RvnjjGckI358NI8BkvHfVWUvziyaU1RxRBTnOLF1e/PbE36TNFbwtsJcBK7m
+hCXViZUUVvc7lkir74ZqhRJ6aqQ1KD2fCrxo6KLh9ruYU1FS8Y8L5tc=
+-----END RSA PRIVATE KEY-----
```

Comment:
this is only used for some advanced integration with Microsoft software from what I read in the docs, we can switch to generate it

Reply:
I am not sure it's wise to have an RSA Private Key exposed like that in our repo. It looks wrong

Reply:
I agree with @zugao 

In addition, a private key should also not be re-used. If collabora needs this, we should generate it on the fly and inject it.

Reply:
I'll add logic to generate it on the fly and keep as secret